ckeditor5_sections.module

```diff
@@ -157,6 +157,61 @@ function ckeditor5_sections_attach_entities(DocumentSection $section) {
   }
 }
 
+/**
+ * Implements hook_delivery_cart_referenced_content().
+ */
+function ckeditor5_sections_delivery_cart_referenced_content(Drupal\Core\Entity\EntityInterface $entity) {
+  if ($entity->getEntityTypeId() === "node") {
```

Comment:
Hi @elistone !
I think we can remove this entity type condition, so that it works for any kind of entity.

Reply:
Removed